standard/documentation-comments.md

```diff
@@ -292,11 +297,15 @@ where
 
 ```csharp
 /// <summary>This is the entry point of the Point class testing program.
-/// <para>This program tests each method and operator, and
+/// <para>
+/// This program tests each method and operator, and
 /// is intended to be run after any non-trvial maintenance has
-/// been performed on the Point class.</para></summary>
-public static void Main() {
-   // ...
+/// been performed on the Point class.
```

Comment:
Clearly written in an era before unit tests were common :) At some point we might want to change this, but not in this PR...

Reply:
Written as #473